### ISSUE TITLE
PLEASE READ BEFORE POSTING

### ISSUE THREAD

**Issue by kprovance** [reduxframework/redux-framework#1975]
Due to a recently influx of posts to our issue tracker in which the required information is absent, we ask you *please* review our posting guidelines before making ANY post.  Posts without this information may be closed unanswered.  Thanks.

## Posting Guidelines for issues and questions ##
When using our Issue Tracker, you may ask questions where you may be a bit lost or need help understanding the documentation. If, however, you find a bug we require you to read and provide the information contained in our [Contributing Guidelines](https://github.com/ReduxFramework/redux-framework/blob/master/CONTRIBUTING.md). If you do not provide this information, we will request it before we can support you.

If you are stuck in some of your own code, or need help with PHP and anything else not Redux specific, we request you purchase some [Premium Support](http://reduxframework.com/extension/premium-support/) and we will be happy to assist you. If we feel the issue is outside of our scope we will suggest you to purchase some [Premium Support](http://reduxframework.com/extension/premium-support/) in order for us to serve you.

**Comment by corradomatt** [comment]
Yes, please do purchase some [Premium Support](http://reduxframework.com/extension/premium-support/) and support this awesome project. If you're using Redux in a theme and making money with it, chipping in a couple of bucks is the right thing to do.

_I'm just a user of Redux, not a developer of it, nor do I make money if you buy something....and no animals were harmed in the posting of this message. thank you_

**Comment by kprovance** [comment]
With our new support protocol, this post is no longer needed.
